src/explore-education-statistics-frontend/src/modules/glossary/GlossaryIndexPage.tsx

```diff
@@ -9,8 +9,11 @@ import React from 'react';
 
 function GlossaryIndexPage() {
   return (
-    <Page breadcrumbs={[{ name: 'Glossary' }]}>
-      <PageTitle title="Education statistics: glossary" />
+    <Page
+      title="Education statistics: glossary"
```

Comment:
Change this to Glossary (the original title was put in as a placeholder by James and should not be trusted as true content). The breadcrumbs/meta should also respect this title.